standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -2276,6 +2276,9 @@ private void create_table_core(final RawStore ms, final CreateTableRequest req)
           + " already exists");
     }
 
+    tbl.setDbName(tbl.getDbName().trim());
```

Comment:
won't it be much more better if the getDbName and the getTableName getter do the trim?

Reply:
The HiveMetaStore.Table.Java is a generated thrift class. I'm not sure how to implement the trim function in the generated classes. But this is actually a valid suggestion. I'll try and see If I can implement your suggestion. Thanks.

Reply:
There is `org.apache.hadoop.hive.metastore.utils.StringUtils.normalizeIdentifier` to take care of the trim, could we reuse this method?